plugins/poll/assets/javascripts/widgets/discourse-poll.js.es6

```diff
@@ -425,6 +425,7 @@ createWidget("discourse-poll-buttons", {
     const closed = attrs.isClosed;
     const staffOnly = poll.results === "staff_only";
     const isStaff = this.currentUser && this.currentUser.staff;
+    const dataExplorerEnabled = this.siteSettings.data_explorer_enabled === true;
```

Comment:
do we need explicit check on true here ?

Reply:
Yeah, the data explorer can be either missing (plugin not installed) or false (plugin disabled)

Reply:
so it will work for a if anyways:

if (false) // not called
if (undefined) // not called

Reply:
cool, fixed